tests/manage/pv_services/test_resource_deletion_during_pvc_pod_creation_and_io.py

```diff
@@ -0,0 +1,283 @@
+import logging
+from concurrent.futures import ThreadPoolExecutor
+import pytest
+
+from ocs_ci.framework.testlib import ManageTest, tier4
+from ocs_ci.ocs import constants
+from ocs_ci.ocs.resources.pvc import delete_pvcs
+from ocs_ci.ocs.resources.pod import get_fio_rw_iops
+from tests import helpers, disruption_helpers
+from tests.fixtures import (
+    create_rbd_storageclass, create_ceph_block_pool,
+    create_cephfs_storageclass, create_rbd_secret, create_cephfs_secret,
+    create_project, create_pvcs
+)
+
+log = logging.getLogger(__name__)
+
+DISRUPTION_OPS = disruption_helpers.Disruptions()
+
+
+@pytest.fixture()
+def test_fixture(request):
+    """
+    Setup and teardown
+    """
+    cls_ref = request.node.cls
```

Comment:
We are no longer going to define fixtures using class references like this. It's yet to be merged but please reference https://github.com/red-hat-storage/ocs-ci/pull/428 for examples on how we are implementing and utilizing fixtures from now on.

Reply:
Thank You @clacroix12 for pointing this out. The test case will be updated as soon as the PR 428 is merged.

Reply:
@jilju @clacroix12  let's not block this PR. It's stale for a more than a week already. Please proceed and we'll adjust it to the new way of fixtures later.